cmd/xaqt/main.go

```diff
@@ -32,7 +33,7 @@ var context *xaqt.Context
 func main() {
 	port := getEnv("XAQT_PORT", "31337")
 
-	compilers := xaqt.ReadCompilers("data/compilers.json")
+	compilers := xaqt.ReadCompilers(filepath.Join(xaqt.DataPath(), "compilers.json"))
```

Comment:
This is a solid default compared to the prior behavior. In the longer run I'd want there to be an optional CLI flag to manually specify this location and/or a REST endpoint to re-specify the compilers details while the server is running.